admin/src/constants/routes.ts

```diff
@@ -0,0 +1,4 @@
+export enum ROUTES {
```

Comment:
better way to use `const`. use `enum` only in cases when this is related to units of measure or similar